From 5e25203fa7c5d81d03e01b0303b18a5410c33c3d Mon Sep 17 00:00:00 2001
From: ming_qian <ming.qian@nxp.com>
Date: Fri, 12 Oct 2018 09:49:21 +0800
Subject: [PATCH 4826/5242] MMFMWK-8274: iMX8QXP_MEK: set default profile to
 baseline

commit  9440d9b70405cd5073679c7e4c44c293cb108646 from
https://source.codeaurora.org/external/imx/linux-imx.git

avoid video mess for 176x144

encode 176x144 with main or high profile, the video may mess

Signed-off-by: ming_qian <ming.qian@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu-encoder-b0/vpu_encoder_ctrl.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/vpu-encoder-b0/vpu_encoder_ctrl.c b/drivers/mxc/vpu-encoder-b0/vpu_encoder_ctrl.c
index c9b6e2f..4833863 100644
--- a/drivers/mxc/vpu-encoder-b0/vpu_encoder_ctrl.c
+++ b/drivers/mxc/vpu-encoder-b0/vpu_encoder_ctrl.c
@@ -213,7 +213,7 @@ static int add_ctrl_h264_profile(struct vpu_ctx *ctx)
 				      V4L2_CID_MPEG_VIDEO_H264_PROFILE,
 				      V4L2_MPEG_VIDEO_H264_PROFILE_HIGH,
 				      0xa,
-				      V4L2_MPEG_VIDEO_H264_PROFILE_MAIN);
+				      V4L2_MPEG_VIDEO_H264_PROFILE_BASELINE);
 	if (!ctrl) {
 		vpu_dbg(LVL_ERR, "add ctrl h264 profile fail\n");
 		return -EINVAL;
-- 
1.7.9.5

